src/main.rs

```diff
@@ -27,16 +29,25 @@ fn main() {
 }
 
 fn eval_print(input: &str) {
+    debug!("input: {:?}", &input);
     let ast = Calc::from_str(input);
+    debug!("AST: {:?}", &ast);
     match ast {
         Ok(ast_node) => {
             let bytecode = &mut vec![];
             Calc::to_bytecode(ast_node, bytecode);
+            debug!("Bytecode: {:?}", &bytecode);
             match Calc::eval(bytecode) {
                 Ok(i) => println!("Result: {}", i),
-                Err(msg) => eprintln!("Evaluation error: {}", msg),
+                Err(msg) => {
+                    error!("Evaluation error: {}", msg);
+                    eprintln!("Evaluation error: {}", msg);
+                }
             }
         }
-        Err(msg) => eprintln!("Evaluation error: {}", msg),
+        Err(msg) => {
+            error!("Evaluation error: {}", msg);
```

Comment:
It's gonna print that anyway, whether or not logging is enabled.

Reply:
yes, I thought that should be expected - this error is the user feedback.
Otherwise no feedback will be given for parsing errors

Reply:
actually:
`error!("Evaluation error: {}", msg);`
will be printed based on the env var
however:
`eprintln!("Evaluation error: {}", msg);`
will be always printed

Reply:
output example:
```
$ RUST_LOG=error  cargo run -- '2+?'
[2023-03-07T11:48:29Z ERROR yaiwr] Evaluation error: Parsing error at line 1 column 3. Repair sequences found:
       1: Insert INT, Delete ?
Evaluation error: Parsing error at line 1 column 3. Repair sequences found:
   1: Insert INT, Delete ?
```
vs
```
$ RUST_LOG=nada  cargo run -- '2+?'
Evaluation error: Parsing error at line 1 column 3. Repair sequences found:
   1: Insert INT, Delete ?
```

Reply:
that's surprising, I expected the undefined `RUST_LOG` to be the same as the non-set level.
removed :point_right: https://github.com/softdevteam/pavel.yaiwr/pull/11/commits/24c4e16f7c76d1964020c351ee1db624d07be536

Reply:
Is that right? Should the error always be printed?

I was expecting you to remove the `error!`, but leave the `eprintln!`

Reply:
I don't see that much difference both will print an error message to stderr.
But, maybe having non-structured logs as user feedback makes more sense.  I set `eprintln!` back :point_right: https://github.com/softdevteam/pavel.yaiwr/pull/11/commits/3c83989b46339beb0bc588f93ae82958d1a8677e

Reply:
That makes more sense to me, as the other way around the user will see no error message at all if logging isn't enabled.